wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,57 +1,83 @@
 # osu! judgement system
 
-## Hit result
-
-| Image | Name | Effect |
-| :-: | :-: | :-- |
-| ![](/wiki/shared/judgement/osu!/hit300.png "300") | 300 | The best achievable result for a hit object. Scoring nothing except 300s in a beatmap will award the player with the SS or SSH [grade](/wiki/Gameplay/Grade). Usually denoted with a blue colour. |
-| ![](/wiki/shared/judgement/osu!/hit300g.png "Geki") | 300 (激) Geki | A variant of the 300 result. Originates from [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan), called Elite Beat! in [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents). Granted upon hitting a 300 on the last element in a combo in which the player has scored only 300s. Getting a Geki will grant a sizable health bonus on top of the standard health increase for a 300. |
-| ![](/wiki/shared/judgement/osu!/hit300k.png "300 Katu") | 300 (喝) Katu or Katsu | A variant of the 300 result. Originates from [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan), called Beat! in [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents). Granted upon hitting a 300 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 300. |
-| ![](/wiki/shared/judgement/osu!/hit100.png "100") | 100 | The second best achievable result for a hit object. Usually denoted with a green colour. |
-| ![](/wiki/shared/judgement/osu!/hit100k.png "100 Katu") | 100 (喝) Katu or Katsu | A variant of the 100 result. Originates from [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan), called Beat! in [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents). Granted upon hitting a 100 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 100. |
-| ![](/wiki/shared/judgement/osu!/hit50.png "50") | 50 | The third best achievable result for a hit object, the last one before a miss. Usually denoted with an orange colour. Scoring a 50 will prevent the appearance of a Katu or a Geki at the combo's end. |
-| ![](/wiki/shared/judgement/osu!/hit0.png "Miss") | Miss | The worst possible result for a hit object. Scoring a Miss will reset the current combo to 0 and will prevent the appearance of a Katu or a Geki at the combo's end. |
+## Judgements
+
+A **judgement** (a.k.a. **hit result**) is the outcome of interacting with a [hit object](/wiki/Gameplay/Hit_object) during its hit window. Score and accuracy are calculated based on which judgements are received.
```

Comment:
this should also go to the index article (`/wiki/Gameplay/Judgement`)

Reply:
.. eh? you can navigate up the file hierarchy using the breadcrumb at the top of the page, so i don't see the point. this is *the* article about judgements[^osu]

[^osu]: (in osu)

Reply:
late comment: what I meant is that the judgement page is currently empty: https://osu.ppy.sh/wiki/en/Gameplay/Judgement, and the description you added to individual game mode judgement pages would fit it perfectly

Reply:
sorta want to address that in a follow-up

1. yeah, it fits (and might end up in there in some form), but for now it'll just cause more duplication than necessary
2. i kinda want the index to describe the "judgement system" in general and explain it in a way that relates it more to gameplay, while the individual articles cover the "judgements" themselves and their requirements

the exact way the info/explanations will be laid out needs a bit more thinking and is a bit more than this pr intends to chew on, which is getting the facts right